templates/masterclasses.html

```diff
@@ -116,7 +116,7 @@ <h2 class="p-heading--3">Previous sessions</h2>
         <div class="u-equal-height">
           {% for session in sprint_sessions %}
           {% with session=session %}
-          {% include "shared/_card--masterclass.html" %}
+          {% include "shared/_card--masterclass-sprint.html" %}
```

Comment:
Are we duplicating the whole file to change the link, or is there another reason?
For the former, we should avoid duplicating it.

Reply:
Thanks for the suggestion. Just made the links reusable blocks.